bin/triggers_check

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+Dir.chdir(File.dirname(__dir__))
+
```

Comment:
Why don't you place this in commit `Add AutoHCK.run` ?

Reply:
Removed with commit 49e76f06c369f0e37aced501a1f3d8d7c01be184.